README.md

```diff
@@ -13,6 +13,32 @@ The _only_  deployable artifact associated with this repository is the
 `requirements-freeze.txt` file. See [datagov-deploy](https://github.com/GSA/datagov-deploy)
 for full configuration in live environments.
 
+The live environment is different than the development environment in a number of ways,
+so it is important to recognize how changes made to the application need to be captured
+so that the application is deployed properly.
```

Comment:
```suggestion
```

Reply:
I kind of like leaving in the idea that the additional work is focused on the goal that the application is deployable.